frontend/app/src/pages/defi/deposits/liquidity/sushiswap/index.vue

```diff
@@ -18,9 +19,21 @@ const secondaryRefreshing = isLoading(secondSection);
 const loading = shouldShowLoadingScreen(section);
 const isEnabled = isModuleEnabled(modules[0]);
 
+const refresh = async (ignoreCache: boolean = false) => {
+  await Promise.all([fetchBalances(ignoreCache), fetchEvents(ignoreCache)]);
+};
+
 onMounted(async () => {
-  await Promise.all([fetchBalances(false), fetchEvents(false)]);
+  await refresh();
 });
+
+const refreshTooltip: Ref<string> = ref(
+  t('helpers.refresh_header.tooltip', {
+    title: t(
+      'navigation_menu.defi_sub.deposits_sub.liquidity_sub.sushiswap'
+    ).toLocaleLowerCase()
+  })
+);
```

Comment:
here also I am not sure ref makes much sense shouldn't computed be the proper choice here.